core/test/gather/gatherers/full-page-screenshot-test.js

```diff
@@ -11,24 +11,21 @@ import FullPageScreenshotGatherer from '../../../gather/gatherers/full-page-scre
 let contentSize;
 /** @type {{width?: number, height?: number, dpr: number}} */
 let screenSize;
-/** @type {{width?: number, height?: number}} */
-let screenshotAreaSize;
 /** @type {string[]} */
 let screenshotData;
 let mockContext = createMockContext();
 
 beforeEach(() => {
   contentSize = {width: 100, height: 100};
   screenSize = {width: 100, height: 100, dpr: 1};
-  screenshotAreaSize = contentSize;
   screenshotData = [];
   mockContext = createMockContext();
   mockContext.driver.defaultSession.sendCommand.mockImplementation((method) => {
     if (method === 'Page.getLayoutMetrics') {
       return {
         cssContentSize: contentSize,
         // See comment within _takeScreenshot() implementation
-        cssLayoutViewport: {clientWidth: screenSize.width, clientHeight: screenSize.height},
+        cssVisualViewport: {clientWidth: contentSize.width, clientHeight: contentSize.height},
```

Comment:
Honestly I think we should just remove this unit test file. It requires us to effectively simulate how chrome handles all these different width/height scales. We're better off using smoke tests for this gatherer IMO.